### ISSUE TITLE
InertiaJS component and Blade component

### ISSUE THREAD

**Issue by abdosaeedelhassan** [inertiajs/inertia-laravel#331]
Hi, i have a big project build with blade component (livewire,alpinejs,laravel blade component), i want to moved this project using Inertiajs but the problem it take long time, i tried put it together but inertia require to render vue component, is there away to ignore rendering vue when rendering using laravel view directive to render blade component?, there is a solution by using two different layout one for inertia and other for blade but this not good for me also it took long time to let comping all project components, need help her please, thank you

**Comment by ajnsn** [comment]
Hey @abdosaeedelhassan 
Below are some related question/issues that could help you with your question. 
TL;DR: Inertia is basically a "router" and "data provider" for modern JavaScript SPA front-ends. You cannot easily use blade and JS components together but you can e.g. keep separate routes for each.

- https://github.com/inertiajs/inertia-laravel/issues/319
- https://github.com/inertiajs/inertia-laravel/issues/219
- https://github.com/inertiajs/inertia-laravel/issues/129
- https://github.com/inertiajs/inertia-laravel/issues/107

**Comment by abdosaeedelhassan** [comment]
@ajnsn thank you, i just have one question, if i used two layout solution, is there away to let inertia component work in blade view (with it shared data)?

**Comment by abdosaeedelhassan** [comment]
I think, solving this issue by doing next

https://github.com/inertiajs/inertia-laravel/issues/107#issuecomment-990550886
